src/ds/d400/d400-factory.cpp

```diff
@@ -1004,6 +1009,9 @@ namespace librealsense
                   dev_info, d400_device::_hw_monitor, get_firmware_logs_command(), get_flash_logs_command() )
             , d400_thermal_tracking( d400_device::_thermal_monitor )
         {
+            if( _fw_version >= firmware_version( 5, 15, 1, 4 ) )
```

Comment:
Can we do the same as we did with the limits?
And keep this is 1 place which ask
`D435i OR D455  && FW > x.x.x.x`
?

Reply:
I'm using 2 functions from d400-motion and this is why it is implemented in the c'tor of D455&D435I which has d400_motion instance